### DISCUSSION TITLE
Viewing masterpassword protected debit/credit cards.

### DISCUSSION THREAD

**Discussion by FHFS** [dani-garcia/vaultwarden#3922] [Q&A] [answered]
I cant view my credit /debit cards in the android app or in the firefox browser extension. I'm wondering if anyone else has this problem.
The webpage works perfectly. Is it because im using bitwarden browser extension / android app?

**Comment by BlackDex** [comment]
You need to go to the right type overviewed.
They should be listed at the top. There you can select card for example.

**Reply by FHFS** [reply]
I was unclear, the android app and browser extension crash when I view my cve for my creditcard or pincode for my debit card. I enter my masterpassword and press ok to view the cve/pincode and it crashes.

**Reply by BlackDex** [reply]
If I'm correct there are/where some issues with some clients and master-password for view option.

**Reply by FHFS** [reply]
So it is an issue with Vaultwarden, not with bitwarden.
Do you know the issue number? I cant find it.

**Reply by stefan0xC** [reply, marked as answer]
If you scroll down a bit, you will find #3915 and #3904 - Just to clarify: it's a client-side issue (see https://github.com/bitwarden/clients/issues/6153 and https://github.com/bitwarden/mobile/issues/2733)
